src/util/errors.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2022-present Mirage Aegis
+ * 
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ * 
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ * 
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+/*
+ * This module contains errors/exceptions that command executions may throw
+ */
+
+/**
+ * An error thrown when trying to find a server member that doesn't exist.
+ * Usually occurs when entering a valid user ID of a user who isn't in the
+ * server a command is executed in
+ */
+export class NoMemberFoundError extends Error {
+    public constructor() {
+        super("No such server member found");
+    }
+}
+
+export class UserIsMemberError extends Error {
+    public constructor() {
+        super("The requested user is a member of the server this command was executed in");
+    }
+}
```

Comment:
These are errors defined by us.
`NoMemberFound` error is intended to be thrown when searching a member who isn't part of the server.
`UserIsMemberError` is intended to be thrown when searching for a user outside the server but is actually a member